packages/charts/src/components/MicroBarChart/MicroBarChart.tsx

```diff
@@ -144,7 +148,13 @@ const MicroBarChart: FC<MicroBarChartProps> = forwardRef((props: MicroBarChartPr
     [props.measure]
   );
 
-  const maxValue = dataset ? Math.max(...dataset?.map((item) => getValueByDataKey(item, measure.accessor))) : 0;
+  const maxValue = useMemo(() => {
+    if (dataset) {
+      const maxDatasetValue = Math.max(...dataset?.map((item) => getValueByDataKey(item, measure.accessor)));
+      return customMaxValue && customMaxValue > maxDatasetValue ? customMaxValue : maxDatasetValue;
```

Comment:
As the user should be in full control over the `maxValue` prop, this can be simplified:
```js
      return props.maxValue ?? maxDatasetValue;
```